analysis/topeft_run2/make_cr_and_sr_plots.py

```diff
@@ -225,27 +226,50 @@ def group_bins(histo,bin_map,axis_name="process",drop_unspecified=False):
 
 # Match a given sample name to whatever it is called in the json
 # Will return None if a match is not found
-def get_scale_name(sample_name,sample_group_map):
+#def get_scale_name(sample_name,sample_group_map):
+#    scale_name_for_json = None
+#    if sample_name in sample_group_map["Conv"]:
+#        scale_name_for_json = "convs"
+#    elif sample_name in sample_group_map["Diboson"]:
+#        scale_name_for_json = "Diboson"
+#    elif sample_name in sample_group_map["Triboson"]:
+#        scale_name_for_json = "Triboson"
+#    elif sample_name in sample_group_map["Signal"]:
+#        for proc_str in ["ttH","tllq","ttlnu","ttll","tHq","tttt"]:
+#            if proc_str in sample_name:
+#                # This should only match once, but maybe we should put a check to enforce this
+#                scale_name_for_json = proc_str
+#    return scale_name_for_json
+#
```

Comment:
Have you verified the code blow gives identical results to this block? If so, you can remove the commented out code here.